enterprise/kernel/src/test/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigratorTest.java

```diff
@@ -101,7 +99,7 @@ public void shouldNotDoActualStoreMigrationBetween3_0_5_and_next() throws Except
     public void detectObsoleteCountStoresToRebuildDuringMigration() throws IOException
```

Comment:
Can we add to this test a verification that all `StoreVersion`s have been verified against?